lib/tools.js

```diff
@@ -661,6 +661,10 @@ module.exports = {
                         err.responseCode = err.responseCode || 404;
                         err.code = 'NoSuchMailbox';
                         break;
+                    case 'OVERQUOTA':
```

Comment:
See the comment in https://github.com/nodemailer/wildduck/pull/542/files#diff-786172e327ab72ee48fd40a2bad6f49f157611bd64d1b253b53f596688ad656bR47

`OVERQUOTA` should not return an error code for mailbox count, but, if used, a disk-size related error.

Reply:
Removed from tools.js